### PR TITLE
Fix logging of body in native-mt

### PR DESCRIPTION
**Pull request by rsinukov**
**Subsystem**
Client/Server

**Motivation**
[Empty body in response using macosx64 target](https://youtrack.jetbrains.com/issue/KTOR-479)

**Solution**
Fixed in coroutines_version=1.3.9-native-mt-2. Added tests to verify

### CODE REVIEW

File: ktor-utils/common/src/io/ktor/util/ByteChannels.kt [hunk 1]
```diff
@@ -15,18 +15,20 @@ private const val CHUNK_BUFFER_SIZE = 4096L
...
-    coroutineScope.launch {
+    GlobalScope.launch {
         try {
             while (!isClosedForRead) {
-                this@split.readRemaining(CHUNK_BUFFER_SIZE).use { chunk ->
-                    listOf(
-                        async { first.writePacket(chunk.copy()) },
-                        async { second.writePacket(chunk.copy()) }
-                    ).awaitAll()
+                supervisorScope {
```
Comment (e5l) [inline ktor-utils/common/src/io/ktor/util/ByteChannels.kt:25]: Does it mean that we changed the `split` behaviour? The fail of single channel led to fail the whole split

**Review by e5l — CHANGES_REQUESTED** [review]
Please respond the comments above

File: ktor-utils/common/src/io/ktor/util/ByteChannels.kt (hunk 1 above)
Comment (rsinukov) [reply ktor-utils/common/src/io/ktor/util/ByteChannels.kt]: reverted changes because the original issue is fixed in coroutines

**Comment by e5l** [comment]
lgtm

**Review by e5l — APPROVED** [review]

**Pull request merged**